Library/Homebrew/rubocops/class_cop.rb

```diff
@@ -82,6 +82,14 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             return
           end
 
+          if test.body.single_line? &&
```

Comment:
Does something like `single_statement?` exist?

Reply:
I'm not aware of one, sadly. Could be wrong because Rubocop is a labyrinth heh, but not to my knowledge.

Reply:
CC @GauthamGoli who may have thoughts. I also think `single_line` is a bit shaky for this.

Reply:
https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/node_pattern.rb
I think this can be used.